hello-server.ts

```diff
@@ -0,0 +1,13 @@
+import { createServer, IncomingMessage, ServerResponse } from "http";
+
+function handler(req: IncomingMessage, res: ServerResponse) {
+  res.writeHead(200, { "Content-Type": "text/plain" });
+  res.end(`Hello, world!`);
+}
```

Comment:
- We define a function handler that will process all incoming requests.
- `res.writeHead(200, { "Content-Type": "text/plain" })`: Sets the HTTP status code to 200 (OK) and specifies that the response is plain text.
- `res.end("Hello, world!")`: Ends the response and sends "Hello, world!" to the client.